logic/transfer.js

```diff
@@ -37,16 +43,26 @@ Transfer.prototype.create = function (data, trs) {
 	trs.recipientId = data.recipientId;
 	trs.amount = data.amount;
 
+	if (data.data) {
+		trs.asset.data = data.data;
+	}
+
 	return trs;
 };
+
 /**
- * Returns send fees from constants.
+ * Returns send fees from constants for transactions without asset and send fees + data fees if asset exists.
```

Comment:
`* Returns send fees from constants for transactions without asset, and send fees + data fees if asset exists.`